week-6/Homework/mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,69 @@
+
+
+
 // Write your code here
+//somehow get images in to that div
+//I can load first image initially with 
+//browser start and next button will load remaining images eventually as button pressed ?
+
+
+let imagesArray = ['image1.jpg', 'image2.jpg', 'image3.jpg', 'image4.jpg', 'image5.jpg', 'image6.jpg', 'image7.jpg'];
```

Comment:
```suggestion
const imagesArray = ['image1.jpg', 'image2.jpg', 'image3.jpg', 'image4.jpg', 'image5.jpg', 'image6.jpg', 'image7.jpg'];
```
I'd consider declaring this variable with the `const` keyword as it doesn't need to be changed over time.

Reply:
Its contents don't need to be changed over time that is